lib/generators/surveyor/templates/config/initializers/simple_form_surveyor.rb

```diff
@@ -1,22 +0,0 @@
-SimpleForm.setup do |config|
```

Comment:
Can we have a task to remove this from our app?

Reply:
https://github.com/user-interviews/rails-server/issues/4175